src/v/cluster/types.h

```diff
@@ -1939,6 +1940,20 @@ struct feature_update_license_update_cmd_data
     operator<<(std::ostream&, const feature_update_license_update_cmd_data&);
 };
 
+struct bootstrap_cluster_cmd_data
+  : serde::envelope<bootstrap_cluster_cmd_data, serde::version<0>> {
+    using rpc_adl_exempt = std::true_type;
+    static constexpr int8_t current_version = 0;
```

Comment:
do we need version field ?

Reply:
I thought that's a requirement for serialization... or is it for ADL serialization only?